.kokoro/presubmit/mongodb.sh

```diff
@@ -22,36 +22,50 @@
 rm -rf */*.log
 rm -rf *-*.yaml
 
-# TODO(franzih): Make these work again
-exit 0
+export NODE_ENV=development
+export DATA_BACKEND="mongodb"
 
-# export NODE_ENV=development
-# export DATA_BACKEND="mongodb"
+# Configure gcloud
+export GCLOUD_PROJECT=nodejs-getting-started-tests
+export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/secrets-key.json
+gcloud auth activate-service-account --key-file "$GOOGLE_APPLICATION_CREDENTIALS"
+gcloud config set project $GCLOUD_PROJECT
 
-# # Configure gcloud
-# export GCLOUD_PROJECT=nodejs-getting-started-tests
-# export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/secrets-key.json
-# gcloud auth activate-service-account --key-file "$GOOGLE_APPLICATION_CREDENTIALS"
-# gcloud config set project $GCLOUD_PROJECT
+# Install Node dependencies
+yarn global add @google-cloud/nodejs-repo-tools
```

Comment:
Instead of installing as global modules, the dependencies should be in `package.json`.